### DISCUSSION TITLE
Where can I find all the fonts available for typst.app?

### DISCUSSION THREAD

**Discussion by wesleyel** [typst/typst#4600] [Q&A] [answered]
Fonts such as `IBM Plex Serif` and `Noto Sans CJK SC` are available by default on typest.app. They are amazing. 

I would like to know if there is a place that lists all the fonts that can be accessed on typest.app.

**Comment by flokl** [comment, marked as answer]
Hi, there is the "Ag" button that displays a list of all fonts available by default.

**Reply by wesleyel** [reply]
![image](https://github.com/user-attachments/assets/6a0de523-9fc1-4dcc-bda1-8675c7c8852e)
Thanks!

**Comment by cskeeters** [comment]
From <https://typst.app/docs/reference/text/text/#parameters-font>:

> In the web app, you can see the list of available fonts by clicking on the "Ag" button. You can provide additional fonts by uploading .ttf or .otf files into your project. They will be discovered automatically. The priority is: project fonts > server fonts.

https://forum.typst.app/t/use-any-font-with-typst-app/6831
